src/example/com/zendesk/maxwell/example/producerfactory/CustomProducer.java

```diff
@@ -25,19 +35,28 @@ public void push(RowMap r) throws Exception
 	{
 		// filtering out DDL and heartbeat rows
 		if(!r.shouldOutput(outputConfig)) {
+			// though not strictly necessary (as skipping has no side effects), we store our position,
+			// so maxwell won't have to "re-skip" this position if crashing and restarting.
```

Comment:
this comment is just a touch mis-leading, we'd prefer to not re-process DDL statements, although I *think* they're idempotent to maxwell

Reply:
In this example producer we ignore all DDL events (`f(!r.shouldOutput(outputConfig))`), so it can re-processes them as many times it wants in case of restarts. I'm not to familiar with the rest of the codebase, so I don't know if choosing to commit our position or not has any side effects.

Rereading the message though, I think it's just unnecessarily confusing. If the producer can safely ignore the message, the example probably shouldn't commit the position, and we can remove that comment to avoid any confusion. And if committing/not committing our position has side effects, then that should be what's highlighted in the comment instead.